.goreleaser.yaml

```diff
@@ -1,49 +1,77 @@
 project_name: kubepug
-release:
-  github:
-    owner: rikatz
-    name: kubepug
+
+env:
+  - COSIGN_YES=true
+
+before:
+  hooks:
+    - go mod tidy
+    - /bin/bash -c 'if [ -n "$(git --no-pager diff --exit-code go.mod go.sum)" ]; then exit 1; fi'
+
+gomod:
+  proxy: true
 
 builds:
   - id: kubepug
     goos:
-    - linux
-    - windows
-    - darwin
+      - linux
+      - windows
+      - darwin
     goarch:
-    - amd64
-    - arm64
-    - arm
-    - s390x
-    - ppc64le
+      - amd64
+      - arm64
+      - arm
+      - s390x
+      - ppc64le
     goarm:
       - 7
     ignore:
-    - goos: windows
-      goarch: arm64
-    - goos: windows
-      goarch: arm
-    - goos: windows
-      goarch: s390x
-    - goos: windows
-      goarch: ppc64le
+      - goos: windows
+        goarch: arm64
+      - goos: windows
+        goarch: arm
+      - goos: windows
+        goarch: s390x
+      - goos: windows
+        goarch: ppc64le
     flags:
-    - -trimpath
+      - -trimpath
     ldflags:
-    - "{{ .Env.LDFLAGS }}"
+      - "{{ .Env.LDFLAGS }}"
     env:
       - CGO_ENABLED=0
       - GO111MODULE=on
-      - GOPROXY=https://proxy.golang.org
     main: .
-    binary: output/kubepug
+    binary: kubepug
```

Comment:
Uh I did changed this on last commit and added output/ to gitignore. Because it was messing with my ignore setup :/ 

Cant it be on output/ ?

Reply:
it put in the path `./dist/kubepug_linux_amd64_v1/output/KUBEPUG` which makes no sense and the dist is already on the gitignore, just making it more clear and we do not need to add that

But i can revert that if you really want

Reply:
ah ok, my problem was that the make build was outputing to root dir, but because I'm an animal! haha

Let's merge this and check how this goes